libs/blocks/global-navigation/utilities/getUserEntitlements.js

```diff
@@ -132,6 +122,7 @@ const getSubscriptions = async ({ queryParams, locale }) => {
  * @param {string} object.format format function, raw or default
  * @returns {object} JIL Entitlements
  */
+// TODO the locale could use the milo format, currently that's an AEM relict.
```

Comment:
Spelling nit: _relic_ 😅

Reply:
According to the internet relict also works https://www.merriam-webster.com/dictionary/relict

Reply:
> In the 16th century, relict meant "an object esteemed and venerated because of association with a saint or martyr," but that meaning is now covered by "relic," a related word that can also refer to something left behind after decay or disappearance. "Relict" was also used to refer to a widow at one time, but now that sense is more or less limited to legal uses. It seems fitting that "relict" has outdated senses; after all, it derives ultimately from the Latin verb relinquere, meaning "to leave behind."

So I'd argue that _relic_ is more adequate for this century and this particular use-case. I'm sure the Milo community will love the grammar debates 😄